tests/integration/pip_parse/.gitignore

```diff
@@ -0,0 +1,4 @@
+# git ignore patterns
```

Comment:
This shouldn't be necessary? The top-level gitignore should also be ignoring these